src/styles/modifiers.scss

```diff
@@ -0,0 +1,79 @@
+* {
+  box-sizing: border-box;
+  }
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid $main-border-color;
+  border-radius: 5px;
+  align-items: center;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    padding-top: 32px;
+    padding-bottom: 40px;
```

Comment:
better to make paddings for .card (for each side) and inside card should be only vertical margins between elements. it is should be on bottom or only top margins.